python/cudf/cudf/dataframe/dataframe.py

```diff
@@ -2734,6 +2734,33 @@ def _create_output_frame(data, percentiles=None):
 
         return output_frame
 
+    def isnull(self):
+        """Identify missing values in a DataFrame.
+        """
+        return DataFrame(
```

Comment:
You should be able to use `_apply_support_method` for this

Reply:
Changed it to use `_apply_support_method`